### ISSUE TITLE
Special processing for genotype name text to enhance downstream UI rendering

### ISSUE THREAD

**Issue by jmcmurry** [monarch-initiative/dipper#209]
Follow-on ticket to Chris' https://github.com/monarch-initiative/monarch-app/issues/754

We currently render genotype names as plain text, escaping the angle brackets.
![screen shot 2015-10-12 at 14 06 51](https://cloud.githubusercontent.com/assets/3658863/10438787/8c105954-70ea-11e5-9df1-7ff1be273d1a.png)

This is okay, but it is desirable to render them as intended eg. as superscripted in http://www.informatics.jax.org/allele/MGI:3894845:
![screen shot 2015-10-12 at 14 12 59](https://cloud.githubusercontent.com/assets/3658863/10438929/6f8b8ab4-70eb-11e5-9f25-b12173ffb0cc.png)

Chris recommends a special field named 'html tag' to avoid complex UI-side handling. In this field the text would be literals that are intended to be rendered as html.

**Comment by nlwashington** [comment]
Because each MOD has a different way of displaying genotypic data, we have a Monarch-y way of regenerating labels, which is what you see.  I believe the original labels are kept as synonyms.
